src/shared/tpm2-util.c

```diff
@@ -3507,36 +3507,42 @@ static int tpm2_build_sealing_policy(
 }
 
 #if HAVE_OPENSSL
-static int tpm2_ecc_curve_from_openssl_curve_id(int curve_id, TPM2_ECC_CURVE *ret) {
+static const struct {
+        TPM2_ECC_CURVE tpm2_ecc_curve_id;
+        int openssl_ecc_curve_id;
+} TPM2_OPENSSL_ECC_CURVE_TABLE[] = {
+        { TPM2_ECC_NIST_P192, NID_X9_62_prime192v1, },
+        { TPM2_ECC_NIST_P224, NID_secp224r1, },
+        { TPM2_ECC_NIST_P256, NID_X9_62_prime256v1, },
+        { TPM2_ECC_NIST_P384, NID_secp384r1, },
+        { TPM2_ECC_NIST_P521, NID_secp521r1, },
+        { TPM2_ECC_SM2_P256,  NID_sm2, },
```

Comment:
doesn't matter, but we'd usually also align the closing `}` on the same column

Reply:
will do